src/server/lib/typedRoutes.ts

```diff
@@ -0,0 +1,16 @@
+import { RoutePaths } from '@/shared/model/Routes';
+import { Router } from 'express';
+
+const router = Router();
+
+export const typedRouter = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  get: (path: RoutePaths, ...handlers: Array<any>) => {
```

Comment:
trying to figure out a way we could avoid the `any` here, could the `Paramters` type  possibly work? https://www.typescriptlang.org/docs/handbook/utility-types.html#parameterstype

Reply:
actually i tried this and it doesn't quite seem to work 😅
might just have to rely on the `any`

Reply:
Cool, I did play around with it too, but couldn't get anything else to work